src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+input {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  width: 100%;
+  background-color: #fff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+::placeholder {
```

Comment:
The ::placeholder selector is too generic and should be targeted to the input elements.